bin/migrate/migrate_cansim_export.py

```diff
@@ -47,81 +47,84 @@
+                    if rec["subject"] in subject_replacements:
+                        rec["subject"] = subject_replacements[rec["subject"]]
+                    else:
+                        print >> sys.stderr, 'Invalid subject "{0}" for {1}'.format(rec["subject"], rec["id"])
+                        continue
+                rec['author_email'] = 'statcan.infostats-infostats.statcan@canada.ca'
+                rec['maintainer_email'] = 'statcan.infostats-infostats.statcan@canada.ca'
+
+                url_en, url_fr = rec.pop('url', None), rec.pop('url_fra', None)
+                rec['program_url'] = {'en':url_en, 'fr':url_fr}
+
+                notes_en = rec.pop('notes','')
+                notes_fr = rec.pop('notes_fra','')
+                rec['notes_translated'] = {'en':notes_en, 'fr':notes_fr}
+
+                data_series_name_en = rec.pop('data_series_name', '')
+                data_series_name_fr = rec.pop('data_series_name_fra', '')
+                rec['data_series_name'] = {'en':data_series_name_en, 'fr':data_series_name_en}
+
+                title_en = rec.get('title', '')
+                title_fr = rec.pop('title_fra', '')
+                rec['title_translated']={'en':title_en, 'fr':title_fr}
+
+                data_series_issue_identification = rec.pop("data_series_issue_identification", "")
+                data_series_issue_identification_fra = rec.pop("data_series_issue_identification_fra", "")
+                rec["data_series_issue_identification"]={
+                        'en':data_series_issue_identification,
+                        'fr':data_series_issue_identification_fra
+                }
+
+                freq = rec.pop('maintenance_and_update_frequency', '')
+                if freq:
+                    rec['frequency'] = freq_dict.get(freq, 'as_needed')
+
+                keywords = rec.pop('keywords','')
+                keywords_fr = rec.pop('keywords_fra','')
+                rec['keywords'] = {'en':[keywords], 'fr':[keywords_fr]}
+
+                rec['collection'] = 'primary'
+                rec['jurisdiction'] = 'federal'
+                rec['geographic_region']= []
+                rec['restrictions'] = 'unrestricted'
+
+                for res in rec['resources']:
+                    name = res.pop('name','')
+                    name_fr = res.pop('name_fra','')
+                    res['name_translated'] = {'en':name, 'fr':name_fr}
+                    lang = res.pop('language', '')
+                    res['language'] = lang_dict.get(lang, [])
+
+                    rformat = res.get('format', '')
+                    if rformat =='HTML':
+                        res['resource_type']='guide'
+                    else:
+                        res['resource_type']='dataset'
+
+                print simplejson.dumps(rec)
+
+        except IOError:
+            print >> sys.stderr, 'Error while reading line.'
+
+    except KeyError:
+        sys.exit(85)
```

Comment:
you don't neet to use two `try:` blocks, just add a second `except` clause

Reply:
actually, this script isn't related to PD so you can ignore it (maybe we should move it out of this directory)